docs/modules/ROOT/pages/ios_faq.adoc

```diff
@@ -10,12 +13,12 @@ Here you can find some of the most frequently asked questions about the ownCloud
 === Compatibility with ownCloud server older than version 10.0
 
 Previous ownCloud iOS app with support for ownCloud server < 10.0 is still available in App Store:  
-https://apps.apple.com/app/id543672169
+{ios-legacy-appstore-url}
 
 === Compatibility with iOS versions older than iOS 12.0
 
 Previous ownCloud iOS app with support for iOS < 12.0 is still available in App Store:  
-https://apps.apple.com/app/id543672169
+{ios-legacy-appstore-url}
```

Comment:
I cant do an easy commit-proposal as it spawns two lines here.
Proposing the following:
`Previous ownCloud iOS app with support for iOS < 12.0 is still available in {ios-legacy-appstore-url}[App Store]`
This makes the link part of the sentence and better readable.

Reply:
fixed with https://github.com/owncloud/ios-app/pull/798/commits/28783c88091138713ac017492fccccda7e21dfd6